src/services/homework/hooks/submissions.js

```diff
@@ -231,23 +231,20 @@ const noDuplicateSubmissionForTeamMembers = (context) => {
 			newTeamMembers = newTeamMembers.filter((teamMember) => !context.data.submission.teamMembers.includes(teamMember.toString()));
 		}
 
-		let toRemove = '';
 		const submissionsForTeamMembers = context.data.submissions.filter((submission) => {
 			for (let i = 0; i < newTeamMembers.length; i += 1) {
 				const teamMember = newTeamMembers[i].toString();
 				if (submission.teamMembers.includes(teamMember)
 						|| (((submission.studentId || {})._id || {}).toString() == teamMember)
 				) {
-					toRemove += (toRemove == '') ? '' : ', ';
-					toRemove += `${submission.studentId.firstName} ${submission.studentId.lastName}`;
 					return true;
 				}
 			}
 			return false;
 		});
 		if (submissionsForTeamMembers.length > 0) {
 			return Promise.reject(new Conflict({
-				message: `${toRemove + ((submissionsForTeamMembers.length == 1) ? ' hat' : ' haben')} bereits eine Lösung abgegeben! Entferne diese${(submissionsForTeamMembers.length == 1) ? 's Mitglied!' : ' Mitglieder!'}`,
+				message: `${submissionsForTeamMembers.length + ((submissionsForTeamMembers.length === 1) ? ' Mitglied hat' : ' Mitglieder haben')} bereits eine Lösung abgegeben! Entferne diese${(submissionsForTeamMembers.length === 1) ? 's Mitglied!' : ' Mitglieder!'}`,
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [This line has a length of 270. Maximum allowed is 120.](https://app.codacy.com/manual/schul-cloud/schulcloud-server/pullRequest?prid=5660867)

Reply:
approved but please fix this first